roles/erpc/templates/erpc.yaml.j2

```diff
@@ -0,0 +1,45 @@
+logLevel: warn
+
+server:
+  listenV4: true
+  httpHostV4: "0.0.0.0"
+  listenV6: false
+  httpPort: 4000
+  aliasing:
+    rules:
+{% if erpc_aliasing_rules %}
+      {{ erpc_aliasing_rules }}
+{% elif not erpc_projects %}
+      - matchDomain: "*"
+        serveArchitecture: "evm"
+        serveChain: {{ erpc_chaines[0].evm_id }}
+        serveProject: "{{ erpc_chaines[0].name }}"
+{% endif %}
+
+projects:
+{% if erpc_projects %}
+  {{ erpc_projects }}
+{% else %}
+{% for chain in erpc_chaines %}
+  - id: {{ chain.name }}
+    upstreamDefaults:
+      rateLimitBudget: {{ chain.limiter | default('default', true) }}
+      type: evm
+      evm:
+        chainId: {{ chain.evm_id }}
+    upstreams:
+{% for node in erpc_endpoint_list %}
+      - endpoint: "{{ node }}/ext/bc/{{ chain.chain_id }}/rpc"
```

Comment:
If we're gonna have /ext/bc/C/rpc then why putting 8545 as default port? Go for 9650 since we're gonna use this role for Avalanche